VideoLocker/src/main/java/org/edx/mobile/task/GetCommentsListTask.java

```diff
@@ -25,8 +25,15 @@ public GetCommentsListTask(@NonNull Context context, @NonNull String responseId,
     }
 
     public Page<DiscussionComment> call() throws Exception {
-        List<String> requestedFields = Collections.singletonList(
-                DiscussionRequestFields.PROFILE_IMAGE.getQueryParamValue());
+        final List<String> requestedFields;
+        {
```

Comment:
> different case for different required fields addition in the list.

That would probably also be in an `if`/`else` statement, or it would be a `switch` statement; all of which can have their own internal blocks.